lib/view/account-info-panel.js

```diff
@@ -24,7 +24,7 @@ class AccountInfoPanel {
     this.view = gui.Container.create()
     this.view.setStyle({
       width: '100%',
-      height: 40,
+      minHeight: 60
```

Comment:
While this value is similar to official app, they have an extra line showing presence while we don't have it in Wey. So I prefer a shorter height for now.